channels/chan_simpleusb.c

```diff
@@ -3197,6 +3197,37 @@ static void _menu_txb(int fd, struct chan_simpleusb_pvt *o, const char *str)
 	return;
 }
 
+/*!
+ * \brief Update the tune settings to the configuration file.
+ * \param filename	The configuration file being updated (e.g. "simpleusb.conf").
+ * \param category	The category being updated (e.g. "12345").
+ * \param variable	The variable being updated (e.g. "rxboost").
+ * \param value		The value being updated (e.g. "yes").
+ * \retval 0		If successful.
+ * \retval -1		If unsuccessful.
+ */
+static int tune_variable_update(const char *filename, struct ast_category *category,
+								const char *variable, const char *value)
+{
+	int	res;
+	struct ast_variable *var;
+
+	res = ast_variable_update(category, variable, value, NULL, 0);
+	if (res == 0) {
+		return 0;
+	}
+
+		/* if we could not find/update the variable, create new */
+		var = ast_variable_new(variable, value, filename);
+	if (var == NULL) {
+		return -1;
+	}
+
+			/* and append */
```

Comment:
Indentation is off in this function now (at least looking at it on GitHub)

Reply:
What you are seeing is the difference between the 4-character tabstops used in the file and the default/typical 8-character tabstops.  GitHub doesn't know what tabstop settings were/are used in the file.